index.js

```diff
@@ -80,6 +83,7 @@ function getBoardActivity(boardId){
 
 			var eventType = boardActions[ix].type;
 			e.emit(eventType, boardActions[ix], boardId);
+			e.emit('actions', eventType, boardActions[ix], boardId);
```

Comment:
Thanks for the contribution. Sorry it took me so long to notice it! What's the purpose of adding this additional event emit?